examples/dt/dcn/service-values.yaml

```diff
@@ -0,0 +1,249 @@
+      backup_ceph_user = openstack
+  cinderVolumes:
+    ceph:
+      customServiceConfig: |
+        [DEFAULT]
+        enabled_backends = ceph
+        [ceph]
+        volume_backend_name = ceph
+        volume_driver = cinder.volume.drivers.rbd.RBDDriver
+        rbd_ceph_conf = /etc/ceph/ceph.conf
+        rbd_user = openstack
+        rbd_pool = volumes
+        rbd_flatten_volume_from_snapshot = False
+        rbd_secret_uuid = CHANGEME
+        rbd_cluster_name = ceph
+        backend_availability_zone = ceph
+    az1:
+      customServiceConfig: |
+        [DEFAULT]
+        enabled_backends = ceph
+        glance_api_servers = http://glance-az1-internal.openstack.svc:9292
+        [ceph]
+        volume_backend_name = ceph
+        volume_driver = cinder.volume.drivers.rbd.RBDDriver
+        rbd_ceph_conf = /etc/ceph/az1.conf
+        rbd_user = openstack
+        rbd_pool = volumes
+        rbd_flatten_volume_from_snapshot = False
+        rbd_secret_uuid = CHANGEME
+        rbd_cluster_name = az1
+        backend_availability_zone = az1
+    az2:
+      customServiceConfig: |
+        [DEFAULT]
+        enabled_backends = ceph
+        glance_api_servers = http://glance-az2-internal.openstack.svc:9292
+        [ceph]
+        volume_backend_name = ceph
+        volume_driver = cinder.volume.drivers.rbd.RBDDriver
+        rbd_ceph_conf = /etc/ceph/az2.conf
+        rbd_user = openstack
+        rbd_pool = volumes
+        rbd_flatten_volume_from_snapshot = False
+        rbd_secret_uuid = CHANGEME
+        rbd_cluster_name = az2
+        backend_availability_zone = az2
+  glance:
+    customServiceConfig: |
+        [DEFAULT]
+        enabled_backends = default_backend:rbd
+        [glance_store]
+        default_backend = default_backend
+        [default_backend]
+        rbd_store_ceph_conf = /etc/ceph/ceph.conf
+        store_description = "RBD backend"
+        rbd_store_pool = images
+        rbd_store_user = openstack
+        rbd_thin_provisioning = True
+    glanceAPIs:
+      default:
+        customServiceConfig: |
+          [DEFAULT]
+          enabled_import_methods = [web-download,copy-image,glance-direct]
+          enabled_backends = ceph:rbd,az1:rbd,az2:rbd
+          [glance_store]
+          default_backend = ceph
+          [ceph]
+          rbd_store_ceph_conf = /etc/ceph/ceph.conf
+          store_description = "ceph RBD backend"
+          rbd_store_pool = images
+          rbd_store_user = openstack
+          rbd_thin_provisioning = True
+          [az1]
+          rbd_store_ceph_conf = /etc/ceph/az1.conf
+          store_description = "az1 RBD backend"
+          rbd_store_pool = images
+          rbd_store_user = openstack
+          rbd_thin_provisioning = True
+          [az2]
+          rbd_store_ceph_conf = /etc/ceph/az2.conf
+          store_description = "az2 RBD backend"
+          rbd_store_pool = images
+          rbd_store_user = openstack
+          rbd_thin_provisioning = True
```

Comment:
Could we add lines (after this one) to enable automatic image format conversion to raw format? It'd make it easier to import the image in raw format to all glance stores without a need to convert the cirros image from qcow2 to raw format with qemu-img tool (which is not available in opentstackclient pod and It makes it more comfortable if once wanna execute for example tempest test since tempestconf has no ability to create an image in multiple glance stores). It makes sense anyway since we use ceph storage as the backend.
The config lines to enable it would be:
```
            [image_import_opts]
            image_import_plugins = ['image_conversion']
            [image_conversion]
            output_format = raw
```

Reply:
done